packages/xdm-deb/xdm-deb.pacsript

```diff
@@ -0,0 +1,12 @@
+name="xdm-deb"
+pkgver="8.0.29"
+pkgrel="1"
+pkgname="Xtreme-Download-Manager"
+gives="xdman"
+replace="xdman"
+homepage="https://xtremedownloadmanager.com"
+arch=('amd64')
+url="https://github.com/subhra74/xdm/releases/download/${pkgver}/xdman_gtk_${pkgver}_amd64.deb"
+pkgdesc="Xtreme Download Manager (XDM) is a powerful tool to increase download speeds up to 500%, save videos from popular video streaming websites, resume broken/dead downloads, schedule and convert downloads.
```

Comment:
First line is way too long. It should be about 80 characters, and anything past that should be on a new line (but still in the quotes).